exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,9 @@
 // Start by creating a variable `message`
 
+let myName = 'Mohammad               '
+let nameLength = myName.length
+let trimName = myName.trim()
+let message = 'My name is '+trimName+' '+'and my name is '+nameLength+' characters long'
```

Comment:
To simplify your code instead of using a new string to create space you van use a space inside of the quotation marks.